src/tf_container/__init__.py

```diff
@@ -11,8 +11,8 @@
 #  express or implied. See the License for the specific language governing 
 #  permissions and limitations under the License.
 
-from tf_container.train import train
+from tf_container.train_entry_point import train
 from tf_container.serve import load_dependencies, transformer
 import tf_container.serve as serve
 
-__all__ = [train, transformer, serve, load_dependencies]
+__all__ = ['train', 'transformer', 'serve', 'load_dependencies']
```

Comment:
Why is this change necessary?

Reply:
__all__ should include list of names instead of values.

From python doc(https://docs.python.org/2/tutorial/modules.html):

'The import statement uses the following convention: if a package’s __init__.py code defines a list named __all__, it is taken to be the list of module names that should be imported when from package import * is encountered. It is up to the package author to keep this list up-to-date when a new version of the package is released. Package authors may also decide not to support it, if they don’t see a use for importing * from their package. For example, the file sound/effects/__init__.py could contain the following code:

__all__ = ["echo", "surround", "reverse"]'